src/data/ACTIONS/root/AST.ts

```diff
@@ -121,8 +121,8 @@ export const AST = ensureActions({
 		mpCost: 1000,
 	},
 
-	ASPECTED_BENEFIC: {
-		id: 835,
+	ASPECTED_BENEFIC: { // Diurnal Variant
+		id: 3595,
```

Comment:
835 is correct, Noct should be 3595 instead of 17000ish.

Reply:
835 is the `applybuff` id, not the `cast` id.

Reply:
Ah the Status, not the Action, oops.